scipy/signal/fir_filter_design.py

```diff
@@ -1141,4 +1058,4 @@ def minimum_phase(h, method='homomorphic', n_fft=None):
         h_temp = ifft(np.exp(fft(h_temp)))
         h_minimum = h_temp.real
     n_out = n_half + len(h) % 2
-    return h_minimum[:n_out]
+return h_minimum[:n_out]
```

Comment:
This needs to be indented. It belongs to the function and a new line is needed at the end.